solutions/observability/incident-management/create-a-degraded-docs-rule.md

```diff
@@ -0,0 +1,161 @@
+---
+navigation_title: Degraded docs
+products:
+  - id: observability
+  - id: cloud-serverless
+applies_to:
+  stack: ga 9.1
+  serverless: ga
+---
+
+# Create a degrade docs rule [degraded-docs-alert]
+
+::::{note}
+
+The **Editor** role or higher is required to create a degraded docs rules. To learn more, refer to [Assign user roles and privileges](/deploy-manage/users-roles/cloud-organization/manage-users.md#general-assign-user-roles).
+
+::::
+
+
+Based on the data found on the [**Data Set Quality**](../data-set-quality-monitoring.md) page, you can create a rule to notify you when the percentage of degraded documents for a specific data view has exceeded a value over a specific time period.
+
+To access this rule from the **Alerts** page:
+
+1. Go to **Alerts**.
+1. Click **Manage Rules** → **Create rule**.
+1. Under **Select rule type**, select **Degraded docs**.
+
+You can also access this rule from the the **Data Set Quality** page:
+
+1. Go to **Stack Management** → **Data Set Quality** or search for "Data Set Quality" using the [global search field](../../../explore-analyze/find-and-organize/find-apps-and-objects.md).
+1. Select a data set name from the data set table.
+1. Select **Actions** in the upper-right corner of the page.
+1. Select **Create rule**.
+
+## Define the conditions [degraded-docs-rule-conditions]
+
+Define the following conditions for your rule:
+
+1. Set the data view you want to create the rule for. If you access the rule from the **Data Set Quality** page, the data view is automatically populated.
+1. Set the threshold percentage that, when exceeded for a period of time, the rule sends an alert.
+1. Specify how long the threshold must be exceeded before and alert is sent.
+1. (Optional) Set one or more **Group alerts by** fields. Every unique value will create an alert.
+1. Set how often to check the rule conditions by selecting a time value and unit under **Rule schedule**.
```

Comment:
@yngrdyn I wasn't sure if this was exactly right. Is this how the `For the last X minutes` field works?

Reply:
yes, that's is how it works